src/backend/partaj/core/models/referral.py

```diff
@@ -130,6 +130,12 @@ class Referral(models.Model):
         help_text=_("Why is this referral urgent?"),
         blank=True,
     )
+    refuse_explanation = models.TextField(
```

Comment:
IMO this should be stored on a new kind of object that is linked on the related referral activity. This will give us way more flexibility with things like for example reopening a closed referral (in the future) or other changes we might need.